tests/lean/binopInfoTree.lean.expected.out

```diff
@@ -20,8 +20,8 @@
                   ===>
                   binop% HAdd.hAdd✝ 1 2
                   • 1 + 2 : Nat @ ⟨5, 7⟩†-⟨5, 12⟩†
-                    • [.] `HAdd.hAdd._@.binopInfoTree._hyg.176 : none @ ⟨5, 7⟩†-⟨5, 16⟩†
-                    • [.] `HAdd.hAdd._@.binopInfoTree._hyg.178 : none @ ⟨5, 7⟩†-⟨5, 12⟩†
+                    • [.] HAdd.hAdd✝ : none @ ⟨5, 7⟩†-⟨5, 16⟩†
+                    • [.] HAdd.hAdd✝ : none @ ⟨5, 7⟩†-⟨5, 12⟩†
```

Comment:
@leodemoura This should make sure committed `Elab.info` output does not break from unrelated changes